addons/google_calendar/models/google_sync.py

```diff
@@ -166,20 +166,67 @@ def _sync_google2odoo(self, google_events: GoogleEvent, default_reminders=()):
 
         return synced_records
 
+    def _google_error_handling(self, result):
+        # We only handle the most problematic errors of sync events.
+        if result.response.status_code in (403, 400):
+            response = json.loads(result.response.text)
+            message = response['error'].get('message')
+            if self._name == 'calendar.event':
+                start = self.start.strftime('%Y-%m-%d at %H:%M')
+                event_ids = self.id
+                name = self.name
+                error_msg = "Error while syncing event: "
```

Comment:
If displayed to the user, it should be translated

Reply:
The error_msg (I will rename it error_log for clarity) is displayed in the logs.
The body variable is translated as it is posted on the event. error_msg has a little more information useful to the admin.